kafka/source/cmd/controller/main.go

```diff
@@ -19,8 +19,8 @@ package main
 import (
 	"log"
 
-	"github.com/knative/eventing-sources/contrib/kafka/pkg/apis"
-	controller "github.com/knative/eventing-sources/contrib/kafka/pkg/reconciler"
+	"github.com/knative/eventing-sources/kafka/source/pkg/apis"
```

Comment:
now that the repo is called `eventing-contrib`, should all these imports be changed to point to `github.com/knative/eeventing-contrib`?